app/tab-servers.R

```diff
@@ -1,44 +1,44 @@
 tab_servers <- tabItem(
   tabName = "cromwell",
-  fluidRow(h2("Manage Your PROOF Based Cromwell Server"), align = "center"),
+  #fluidRow(h2("Manage Your PROOF Server"), align = "center"),
   fluidRow(
     align = "left",
     box(
-      width = 12, solidHeader = FALSE, status = "info",
+      width = 12, solidHeader = FALSE, status = "warning",
       collapsed = FALSE,
-      title = "Start/delete your Cromwell Server",
-      p(strong("Note"), " stopping your server requires making sure you mean it :)"),
-      br(),
-      br(),
+      title = "Manage your PROOF Server",
+      p("Note: Hover over the ", icon("question-circle"), " icons to get more information about each item."),
+      h4("Get a PROOF Server Running"),
+      p("If you have a PROOF server running, 'Job Status' should indicate 'RUNNING'.  If not, click the 'Start a PROOF server' button."),
+      uiOutput("proofStatusJobStatus"),
       actionButton(
-        inputId = "cromwellStart",
-        label = "Start",
-        icon = icon("play"),
-        class = "btn-primary btn-lg "
+          inputId = "cromwellStart",
+          label = "Start a PROOF Server",
+          icon = icon("play"),
+          class = "btn-success"
       ),
-      actionButton(
-        inputId = "cromwellDelete",
-        label = "Delete",
-        icon = icon("stop"),
-        class = "btn-warning btn-lg"
-      )
-    )
-  ),
-  fluidRow(
-    align = "left",
-    box(
-      width = 12, solidHeader = FALSE, status = "info",
-      collapsed = FALSE,
-      title = "Status",
-      p("Hover over the ", icon("question-circle"), " icon to get more information about each item"),
-      h4("Server information"),
-      uiOutput("proofStatusJobStatus"),
+      
+      br(),
+      br(),
+      h5("Current PROOF server information (if live)"),
       uiOutput("proofStatusWorkflowLogDir"),
       uiOutput("proofStatusScratchDir"),
       uiOutput("proofStatusServerTime"),
       uiOutput("proofStatusSlurmJobAccount"),
       br(),
-      h4("Troubleshooting"),
+      br(),
+      h4("Stop your PROOF Server"),
```

Comment:
Do we want to say "PROOF server" when actually that's more accurately the server that serves the PROOF API? Or is is shortand for "PROOF based Cromwell server" and if so, is it okay to use that shorthand?

Reply:
PROOF based Cromwell Server is long..... aint it?

Reply:
Okay proof it is